distribution/deb/src/main/packaging/init.d/elasticsearch

```diff
@@ -70,6 +70,10 @@ if [ -f "$DEFAULT" ]; then
 	. "$DEFAULT"
 fi
 
+if [ "$ES_USER" != "elasticsearch" ] || [ "$ES_GROUP" != "elasticsearch" ]; then
```

Comment:
Right, but if they change `/etc/default/elasticsearch` *after* installation, we should still warn, no?